### PR TITLE
D3ASIM-3667: Create Future market settings class

### PR DESCRIPTION
**Pull request by Hassan754**

### CODE REVIEW

File: d3a_interface/constants_limits.py [hunk 1]
```diff
@@ -64,6 +64,10 @@ class SettlementMarketSettings:
         ENABLE_SETTLEMENT_MARKETS = False
         RELATIVE_STD_FROM_FORECAST_FLOAT = 10.0
 
+    class FutureMarketSettings:
+        # Duration between clearing in future markets
+        FUTURE_MARKET_CLEARING_INTERVAL = 15  # in minutes
```
Comment (spyrostz) [inline d3a_interface/constants_limits.py:69]: As commented in the other PR, I would prefer FUTURE_MARKET_CLEARING_INTERVAL_M or FUTURE_MARKET_CLEARING_INTERVAL_MINUTES in order to clarify the unit of the constant.

**Review by spyrostz — APPROVED** [review]
One comment, LGTM

**Review by spyrostz — APPROVED** [review]
LGTM

**Review by fievelk — APPROVED** [review]

File: d3a_interface/constants_limits.py (hunk 1 above)
Comment (faizan2590) [reply d3a_interface/constants_limits.py]: Whats concluded here?

Comment (faizan2590) [reply d3a_interface/constants_limits.py]: Nevermind, just saw that its resolved via https://github.com/gridsingularity/gsy-framework/pull/285/commits/18e7081a82a7650e8cd4ef63938cf5a65205cf08

Comment (Hassan754) [reply d3a_interface/constants_limits.py]: @faizan2590 I went for `FUTURE_MARKET_CLEARING_INTERVAL_MINUTES`

**Pull request merged**